src/components/Common/CommunityCard/index.tsx

```diff
@@ -20,8 +20,8 @@ const CommunityCard: React.FC<CommunityCardProps> = ({
   return (
     <div className="card flex-column justify-center align-center">
       <div className="text-center">
-        <div className="font-32 mb16">{title}</div>
-        <div className="font-16 gray mb16">{description}</div>
+        <div className="font-2xlarge mb-base">{title}</div>
```

Comment:
The colour here does not seem very visible at the moment, not sure if you want to resolve it in this PR but feel free to change it in another PR in the future!